packages/components/accordion/src/Accordion.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { Box } from '@contentful/f36-core';
+import getStyles from './Accordion.styles';
+
+export interface AccordionProps {
+  /**
+   * Specify the alignment of the chevron inside the accordion header
+   */
```

Comment:
Maybe add the default value
```suggestion
   * @default end
   */
```

Reply:
will, do, thanks!